app/controllers/test/components/mentoring/discussion_post_list_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Test::Components::Mentoring::DiscussionPostListController < Test::BaseController
+  def show
+    @discussion = Solution::MentorDiscussion.find(params[:discussion_id])
+    @iteration = Iteration.find(params[:iteration_id])
```

Comment:
Both of these need to use uuid and need rescues and `render_404` blocks.

Reply:
This is just a test controller.